File: workspace/people/p02-site-reliability-engineer/home/documents/operations/runbooks/automation/2026-07-20/source/2026-07-20-reliability-runbook-database-failover-automation-3.py
"""Northstar Transit Cloud; Checkout SVC-17 incident context; 2026-07-13; error budget 2.4%; INC-260713."""
SERVICE = "Identity SVC-23"
INCIDENT = "INC-260713"
TOPIC = "database_failover"
SPINE_IDS = ['org.ntc', 'svc.identity.svc23', 'rel.inc260713.timeline', 'date.2026-07-20', 'term.no-kio-claim']
FINGERPRINT = "37045fdaf434b7f7"

def assess_failover_evidence(status: str) -> bool:
    return status == 'reviewed'

File: workspace/people/p02-site-reliability-engineer/home/documents/operations/runbooks/automation/2026-07-20/source/2026-07-20-reliability-runbook-database-failover-automation-3.rs
// Northstar Transit Cloud | Checkout SVC-17 | 2026-07-13 | error budget 2.4% | INC-260713
pub const TOPIC: &str = "database_failover";
pub const FINGERPRINT: &str = "b62c4ced51b15dcb";
pub fn failover_evidence_reviewed(status: &str) -> bool { status == "reviewed" }
